backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointService.java

```diff
@@ -135,7 +152,11 @@
     @APIResponse(responseCode = "204", description = "The integration has been deleted", content = @Content(schema = @Schema(type = SchemaType.STRING)))
     public Uni<Response> deleteEndpoint(@Context SecurityContext sec, @PathParam("id") UUID id) {
         RhIdPrincipal principal = (RhIdPrincipal) sec.getUserPrincipal();
-        return resources.deleteEndpoint(principal.getAccount(), id)
+        return resources.getEndpoint(principal.getAccount(), id)
```

Comment:
It looks like this could be replaced with the new `resources.getEndpointTypeById` method.